### PR TITLE
Remove redundant code

### PR DESCRIPTION
**Pull request by mkondratek**
Just removing a lot of redundant code + minor code improvements & typos fixes.

## Test plan
- CI passes
- plugin can be built and run

### CODE REVIEW

File: src/test/kotlin/com/sourcegraph/cody/chat/PromptHistoryTest.kt [hunk 1]
```diff
@@ -0,0 +1 @@
+
```
Comment (pkukielka) [inline src/test/kotlin/com/sourcegraph/cody/chat/PromptHistoryTest.kt]: ? :)

File: src/test/kotlin/com/sourcegraph/cody/agent/protocol/PositionTest.kt [hunk 2]
```diff
@@ -14,6 +15,16 @@ class PositionTest : BasePlatformTestCase() {
     myFixture.openFileInEditor(file)
   }
 
+  override fun tearDown() {
```
Comment (pkukielka) [inline src/test/kotlin/com/sourcegraph/cody/agent/protocol/PositionTest.kt]: Unfortunately that won't help, I was testing it on my PR.

**Comment by odisseus** [comment]
No new code = no new bugs, right..? Right?

File: src/test/kotlin/com/sourcegraph/cody/chat/PromptHistoryTest.kt (hunk 1 above)
Comment (mkondratek) [reply src/test/kotlin/com/sourcegraph/cody/chat/PromptHistoryTest.kt]: just a missing newline added automatically

File: src/test/kotlin/com/sourcegraph/cody/agent/protocol/PositionTest.kt (hunk 2 above)
Comment (mkondratek) [reply src/test/kotlin/com/sourcegraph/cody/agent/protocol/PositionTest.kt]: reverted then

**Comment by mkondratek** [comment]
> No new code = no new bugs, right..? Right?

less code less bugs 😄

File: src/test/kotlin/com/sourcegraph/cody/chat/PromptHistoryTest.kt (hunk 1 above)
Comment (odisseus) [reply src/test/kotlin/com/sourcegraph/cody/chat/PromptHistoryTest.kt]: Why is this file needed at all?

**Review by odisseus — DISMISSED** [review]

File: src/test/kotlin/com/sourcegraph/cody/chat/PromptHistoryTest.kt (hunk 1 above)
Comment (mkondratek) [reply src/test/kotlin/com/sourcegraph/cody/chat/PromptHistoryTest.kt]: ah, sorry - my bad. I misunderstood the diff

Comment (mkondratek) [reply src/test/kotlin/com/sourcegraph/cody/chat/PromptHistoryTest.kt]: removed

**Review by odisseus — APPROVED** [review]

**Pull request merged**